var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -79,4 +86,11 @@ def cmake_args(self):
             options.append('-DCUDA_TOOLKIT_ROOT_DIR:STRING=' +
                            self.spec['cuda'].prefix)
 
+        if self.spec.variants['simd'].value != 'AUTO':
+            options.append('-DGMX_SIMD:STRING=' +
+                           self.spec.variants['simd'].value)
+
+        if '-rdtscp' in self.spec:
+            options.append('-DGMX_USE_RDTSCP:BOOL=OFF')
```

Comment:
I would prefer adding an explicit case for `ON` instead on relying on Gromacs' default values.